manim/mobject/svg/drawings.py

```diff
@@ -55,6 +55,7 @@
 from ...utils.space_ops import angle_of_vector
 from ...utils.space_ops import complex_to_R3
 from ...utils.space_ops import rotate_vector
+from ...utils.color import YELLOW, WHITE, DARK_GREY, MAROON_B, PURPLE, GREEN, BLACK, LIGHT_GREY, GREY, BLUE_B, BLUE_D
```

Comment:
Why not just import C? And use C.WHITE, etc.

Reply:
also possible.
But I wanted to keep it consistent with the others, so I chose this way
Another option would be to say in this example `from ...utils.color import * `